packages/config/config/devices/0x031e/lzw45.json

```diff
@@ -0,0 +1,301 @@
+{
+	"manufacturer": "Inovelli",
+	"manufacturerId": "0x031e",
+	"label": "LZW45",
+	"description": "Light Strip",
+	"devices": [
+		{
+			"productType": "0x000a",
+			"productId": "0x0001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic_Set",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Switch_MultiLevel_Set",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Switch_MultiLevel_Start/Stop",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Number of Pixels",
+			"description": "When individually addressable LEDs are used, this parameter tells the controller the number of pixels that are attached. \n      0: Automatic recognition of pixels\n      Range 1 - 130: Set the fixed value of the pixel bit\n      Default: 0",
```

Comment:
Remove the `\n` and `Default 0` from the description

Reply:
And the ranges. If there is a special meaning of one value, it can stay in the description. The rest should be addressed as `minValue`, `maxValue` and `default`.